auth/api/iam/localize.go

```diff
@@ -0,0 +1,28 @@
+package iam
+
+import (
+	"strings"
+)
+
+// localize selects from a range of localized values based on the accept-language header, returning
+// a default value if no localized value matches the languages specified in the accept-language header.
+func localize(defaultValue string, localizedValues map[string]string, acceptLanguageHeader string) string {
+	// Iterate over the accepted languages specified by the user-agent
+	for _, acceptedLanguage := range strings.Split(acceptLanguageHeader, ",") {
+		// Trim any whitespace from the language specification
+		acceptedLanguage = strings.TrimSpace(acceptedLanguage)
+
+		// Split the language and any associated q-factor value (e.g. "en;q=0.5") where the q-factor specifies
```

Comment:
does anyone actually implement that properly... talking about elaborate specs

Reply:
Yea this is asking a lot of implementers. It wouldn't be *too* insane to create a map of decimal values and then sort the keys based on them but what would you put as a default value when the q-factor is missing?